test/passing/gen/gen.ml

```diff
@@ -77,9 +81,12 @@ let emit_test test_name setup =
   in
   let extra_deps = String.concat " " setup.extra_deps in
   let enabled_if_line =
-    match setup.minimum_ocaml_version with
-    | None -> ""
-    | Some v -> Printf.sprintf "\n (enabled_if (>= %%{ocaml_version} %s))" v
+    match setup.minimum_ocaml_version, setup.enabled_if with
```

Comment:
There's 6 uses of `minimum-ocaml-version`. Replacing it with `enabled-if` may be a smaller patch and would remove code.